autofit/mapper/prior_model/abstract.py

```diff
@@ -248,7 +251,7 @@ def _with_paths(
 
     def with_paths(
             self,
-            paths: List[Tuple[str]]
+            paths: List[Tuple[str, ...]]

```

Comment:
I am getting lint errors when there is more than 1 string in my tuple specifying paths.

Reply:
Yeah the ... should fix that